Assignment 2 conditional structure

Katsiff opened the pull request.

Katsiff committed:
Assignment 2 conditional structure

ninjapanzer commented:
:shipit: :+1:

garyanewsome commented:
Very nice program. If I were to make one suggestion, I may have checked the Y and E options against both upper and lower case. Perhaps creating a dual case, don't know if that is the right terminology, lol...
Case 'E':
Case 'e':
~code stuff~
break;
That should allow both upper and lower case 'E's" to serve as an exit.

Katsiff commented:
Thank you, @garyanewsome  . I had considered that last night and must have forgotten to or been too tired to delve into it. I will take a look tomorrow. I have been reading and practicing nearly nonstop since Monday to catch up. Spent last Thursday -Sunday on my other classes. It's no fun getting behind!

ninjapanzer commented on siffrinnassignment2conditions/src/siffrinnassignment2conditions/Siffrinnassignment2conditions.java:
```diff
@@ -0,0 +1,79 @@
+                case "Y":
+                    System.out.println("\nSorry it's not available right now.\n");
+                    break;
+                case "E":
```
another way to avoid having to define a case for each the difference between `E` and `e` is to do a `.toLowerCase()` on your input at the very top and then just assuming everything is lowercase. This is a common real world optimization.

Katsiff committed:
changes 

I copied my changed file into this directly on the web because I have tried multiple times to push changes through my branch unsuccessfully. I have deleted and remade files repeatedly. Hoping this works.
